host_vars/galaxy-qa2.galaxy.cloud.e-infra.cz/vars.yml

```diff
@@ -5,7 +5,6 @@ csnt_brand: QA2-TEST
 # csnt_enable_account_interface: false
 csnt_log_level: DEBUG
 csnt_enable_notification_system: true
-csnt_registered_user_jobs_limit: 1000
```

Comment:
is there a reason to remove this?

Reply:
It was for development and testing purposes, it is tied to QA2 host so not sure if we need to keep it.

Reply:
I set it for both QAs as a reasonable default for testing instances -- so we avoid unexpectedly stalled job queues while testin.